alerta/views/alerts.py

```diff
@@ -649,6 +650,9 @@ def update_note(alert_id, note_id):
 
     update = request.json
     update['user'] = g.login
+    if update.get('text', ''):
+        alert, note_text = process_note(alert, update['text'])
+        update['text'] = note_text
```

Comment:
Can you move the above 3 lines to below the `write_audit_trail.send()`? And adjust to look like ...

```
    _, update['text'] = process_note(alert, update['text'])
    updated = note.update(**update)
    if updated:
        return jsonify(status='ok', note=updated.serialize)
    else:
        raise ApiError('failed to update note', 500)
```
I think it should run the hook whether there is text or not. Someone might want to add text if missing, for example.

Reply:
Fix done (I also changed function signature as `None` can be passed) , if you're ok I can rebase and squash

Reply:
Nice. Thanks! No need to rebase/squash. I'll merge as is.